api/src/main/java/pl/kolendateam/dadcard/feats/entity/CharacterFeat.java

```diff
@@ -123,5 +123,189 @@ public boolean checkPrerequisite(
         return checkEvry==ckeckTot;
 
     }
+
+    public String wildShape(int levelOfFeat){
+        String stringFeatSpecial = "";
+
+        if(levelOfFeat < 4){
```

Comment:
fajniej wygladal by switch.  Albo returnowal bym stringFeatSpecial w ifach bo jak levelOfFeat  równe 5 to nie ma sesu sprawdzac czy równe 9

Reply:
zrobiłem na returnach, masz racje